content/blog/webdev-tech.md

```diff
@@ -0,0 +1,191 @@
++++
+title = "WebDev Tech"
+date = 2025-02-26
+draft = false
+authors = ["Blain Smith"]
+
+[taxonomies]
+tags = ["Engineering"]
+
+[extra]
+feature_photo = ""
+feature_photo_alt = ""
++++
+
+At Limeleaf we chose to specialize in Go and Rust for our clients and we've covered that topic as to why we made that choice. However, since we're getting into building our own products we need to be able to develop applications for the web too.
+
+<!-- more -->
+
+## TL;DR
+
+- Use [`templ`][0] in Go and [`maud`][1] for Rust to do server-side HTML rendering
+- Use [`net/http`][2] in Go and [`axum`][3], [`actix`][4], [`rocket`][5], or whatever flavor Rust for HTTP handling and routing
+- Start with and try to stay with [SQLite][6] and migrate to [PostgreSQL][7] if needed
+
+---
+
+Go and Rust are great systems programming languages for networking and high performance services, but not many folks think to use them traditional web applications that render HTML and handle form data. Most modern web applications subscribe to the single page application (SPA) model where there exists some backend REST or GraphQL API and a completely separate frontend application written in JavaScript with some popular framework like React or Vue. 
+
+We think for the products we're building and for the open web these choices are wrong.
```

Comment:
This is fine if we want to be bold, but I would word this more softly. Like "We think that for our products and building for the open web, there are better choices"

Reply:
"...choices are overkill." :question:

Reply:
Nice, thanks. Pushed changes.